mungegithub/features/repo-updates.go

```diff
@@ -297,24 +313,30 @@ func peopleForPath(path string, people map[string]sets.String, leafOnly bool, en
 	return out
 }
 
-// LeafAssignees returns a set of users who are the closest assignees to the
+// LeafApprovers returns a set of users who are the closest approvers to the
 // requested file. If pkg/OWNERS has user1 and pkg/util/OWNERS has user2 this
 // will only return user2 for the path pkg/util/sets/file.go
-func (o *RepoInfo) LeafAssignees(path string) sets.String {
-	return peopleForPath(path, o.assignees, true, o.EnableMdYaml)
+func (o *RepoInfo) LeafApprovers(path string) sets.String {
+	return peopleForPath(path, o.approvers, true, o.EnableMdYaml)
 }
 
-// Assignees returns a set of users who are the closest assignees to the
+// Approvers returns a set of users who are the closest approvers to the
```

Comment:
Not your fault, but can you fix this comment? Approvers() and Reviewers() return ALL of them, not just the closest.